contracts/extensions/Bank.sol

```diff
@@ -422,32 +422,48 @@ contract BankExtension is DaoConstants, AdapterGuard, IExtension {
 
     /**
      * @notice Creates a new amount checkpoint for a token of a certain member
+     * @dev Reverts if the amount is greater than 2**64-1
      * @param member The member whose checkpoints will be added to
-     * @param tokenAddr The token of which the balance will be changed
+     * @param token The token of which the balance will be changed
      * @param amount The amount to be written into the new checkpoint
      */
     function _createNewAmountCheckpoint(
         address member,
-        address tokenAddr,
+        address token,
         uint256 amount
     ) internal {
-        uint32 nCheckpoints = numCheckpoints[tokenAddr][member];
-        require(amount < type(uint160).max, "too big of a vote");
-        uint160 newAmount = uint160(amount);
+        bool isValidToken = false;
+        if (availableInternalTokens[token]) {
```

Comment:
the logic seems good to me here. But the check should be uint88.max for internal tokens and unt160.max for external tokens